sig-ml-developer-experience/OWNERS

```diff
@@ -1,13 +1,20 @@
 approvers:
+- andrewballantyne
 - erwangranger
+- harshad16
 - jkoehler-redhat
 - kywalker-rh
-- romeokienzler
 
 reviewers:
+- andrewballantyne
+- kywalker-rh
+- harshad16
 - goern
 - LaVLaS
 - lucferbux
+
+emeritus_approvers:
+- romeokienzler
 - samuelvl
```

Comment:
Probably could just delete them -- they have not been active for a long while.

Reply:
`emeritus_approvers` is for people who were approvers and no longer active.
would like me to remove them completely off the chart ?

Reply:
Yeah there is no reason to keep such details at this stage. We are effectively starting fresh here.